### ISSUE TITLE
Practice Session 7 #Q.4

### ISSUE THREAD

**Issue by giiiithub** [goldshakil/ElectricalProgrammingCourse2020#198]
I considered the input is one sentence. After I made a code for Q.4, I thought the input is not one sentence but several sentences.
Should I consider that case?
And We learned about structure little bit in lecture 10.
Then can I use structure which is not learned in lecture 10?
ex) ptr<-name is learned but ptr.name is not learned.
Thanks for reading. :)

**Comment by goldshakil** [comment]
Yes you can have one long paragraph of multiple sentences and you can use structures although they are not required to do this problem but your code will look better.

typedef struct word
{
char data[];
int repetition;

} word;
